internal/versions/common.go

```diff
@@ -160,9 +160,15 @@ func validateReleaseImageForRHCOS(
 				}
 			}
 		}
+		if swag.ContainsStrings(releaseImage.CPUArchitectures, cpuArchitecture) && !swag.ContainsStrings(availableOCPVersions, swag.StringValue(minorVersion)) {
+			availableOCPVersions = append(availableOCPVersions, swag.StringValue(releaseImage.OpenshiftVersion))
+		}
 	}
-
-	return errors.Errorf("The requested RHCOS version (%s, arch: %s) does not have a matching OpenShift release image", *rhcosVersionPtr, cpuArchitecture)
+	ocpVersionAdvice := fmt.Sprintf("There are no OCP versions available in release images for arch %s.", cpuArchitecture)
+	if len(availableOCPVersions) > 0 {
+		ocpVersionAdvice = fmt.Sprintf("The following OCP versions are available in release images for arch %s please choose your OCP version to match one of these: %s", cpuArchitecture, strings.Join(availableOCPVersions, ","))
```

Comment:
I would change it to something like - `These are the OCP versions which a matching release image has been found for them`